### ISSUE TITLE
Better explanation needed

### ISSUE THREAD

**Issue by henno** [zielu/GitToolBox#69]
I've spent 15 minutes trying to figure out how GitToolBox should help me becoming aware that I should update the project and I still don't understand. 

First I was under the impression that I get a popup of some kind telling me that it's time to do the update. Then I (mis)understood that the changed files are marked somehow in the Project files sidebar (⌘1, or Alt+1 on a PC). Then I figured that the (remotely) changed lines are somehow marked in the file. But no and no and no.

I also spent 10 minutes figuring out the settings. Specifically, what those various numbers exactly mean when Display mode is set to Text, what is the point of showing only one option in the dropdown of "Update project action", what does "Behind tracker" and "Show project view decoration" options actually do? 

It would be very helpful for newcomers like me now if you added either a Youtube video or a moving GIF animation to README.md explaining how the plugin works.

FWIW, I'm using Phpstorm.

**Comment by zielu** [comment]
Hi, could you take a look at description in Settings > Plugins > GitToolBox and tell me if it contains enough information ?

Nice idea with gifs, I'll look into it. 

In the meantime I'll add tooltips and descriptions for options in Settings to make them clearer.

**Comment by henno** [comment]
I took a look  and saw the tooltips but I'm still confused:
- "Behind tracker": What kind of notification should I expect? I tested by making a commit in Bitbucket website and leaving PHPStorm open. Nothing changed so I left the computer by itself, switched on Caffeine app and went to lunch. When I got back, I saw the tiny green (1) at the bottom right corner of PHPStorm window. I expected something more visible like a small popup which I occasionally see in PHPStorm. I would never notice such tiny little icons on the statusbar.
- "Project view" decoration: Again, even though I have the plugin installed for 9 days, I never noticed anything particular in Project View window. Now with today's test I actually see "master 1^" on the project name line. This again is something I will never notice while I am working. I expected to see that the file that has been modified at remote tracking branch would be in some other color (yellow?) or have some additional icon in front of it or something like that. Or:
![alt text](https://www.upload.ee/image/7939989/Screen_Shot_2018-01-26_at_13.42.38__.png "Example")
